<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Turbulence Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLWidget" name="displayWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="interactionModeGroupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Interaction Mode</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="selectRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Selection</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="exploreRadioButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Explore</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="loadParamsGroupBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Load Parameters File</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="loadParamsButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="loadParamsLineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="volZoomGroupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Volume Zoom Level</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QSlider" name="volZoomSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>160</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="displayOptionsGroupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Display Options</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="volRendCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume Rendering</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="glyphCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Glyphs</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="streamCheckBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Streamlines</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="glyphTypesGroupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Types of Glyphs</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="superquadRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Superquadrics</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="arrowRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Arrow Glyphs</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="magnificationGroupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Magnification</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="magZoomLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Zoom</string>
     </property>
    </widget>
    <widget class="QSlider" name="magZoomSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="magRadiusLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Radius</string>
     </property>
    </widget>
    <widget class="QSlider" name="magRadiusSlider">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <zorder>magZoomLabel</zorder>
    <zorder>magZoomSlider</zorder>
    <zorder>magRadiusLabel</zorder>
    <zorder>magRadiusLabel</zorder>
    <zorder>magRadiusSlider</zorder>
   </widget>
   <widget class="QGroupBox" name="glyphSliceGroupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Arrow Glyphs Axis Slice</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="xSliceCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ySliceCheckBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="zSliceCheckBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QSlider" name="xSlicePosSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="zSlicePosSlider">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="ySlicePosSlider">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTensor">
    <property name="title">
     <string>Tensor</string>
    </property>
   </widget>
   <addaction name="menuTensor"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>streamCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Modes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>578</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>glyphCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Modes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volRendCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Modes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volZoomSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>Zoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exploreRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>magnificationGroupBox</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Modes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>superquadRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>arrowRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Modes()</slot>
  <slot>Zoom()</slot>
  <slot>Change()</slot>
 </slots>
</ui>
